lib/active_triples/data_model.rb

```diff
@@ -0,0 +1,20 @@
+module ActiveTriples
+  ##
+  # Super class which provides a simple property DSL for defining property ->
+  # predicate mappings.
+  class DataModel
```

Comment:
The open-ended property interface bothers me too, but I'm scared to lock it down when we know projects like ActiveFedora have arbitrary metadata they'd like to define for the property relationship.